conversation_visitor_identity.go

```diff
@@ -0,0 +1,34 @@
+package hubspot
+
+const (
+	visitorIdentificationBasePath = "/conversations/v3/visitor-identification"
+)
+
+type IdentificationTokenResponse struct {
+	Token string `json:"token"`
+}
+
+type IdentificationTokenRequest struct {
+	FirstName string `json:"firstname"`
+	LastName  string `json:"lastname"`
+	Email     string `json:"email"`
+}
+
+type VisitorIdentificationService interface {
+	GenerateIdentificationToken(option IdentificationTokenRequest) (*IdentificationTokenResponse, error)
+}
+
+type VisitorIdentificationServiceOp struct {
```

Comment:
I want the struct to have a client and an api base path.
(e.g. https://github.com/belong-inc/go-hubspot/blob/main/contact.go#L21)